CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+`terra-docker` consists of docker images that are compatible with the notebooks service in Terra called Leonardo. We follow the naming convention of `terra-[application]-[image_name]` (for example `terra-jupyter-aou`).
+
+## Opening a PR
+
+We do our work on a feature branch based on master. When you open a pull request, add the JIRA issue number to the PR title. This will make a reference from JIRA to the GitHub issue. Briefly describe your changes in the description section.
```

Comment:
we expect external contributors too other than broad. Maybe add something like indicating no need to care about JIRA if you're from outside broad?

Reply:
yeah I can add that. I can also maybe add a sentence saying "there's still more to come on external contributions"